packages/mdc-ripple/constants.js

```diff
@@ -53,4 +53,5 @@ export const numbers = {
   UNBOUNDED_TRANSFORM_DURATION_MS: 200,
   PADDING: 10,
   INITIAL_ORIGIN_SCALE: 0.6,
+  DEACTIVATION_TIMEOUT_MSL: 200,
```

Comment:
looks like this should be named `DEACTIVATION_TIMEOUT_MSL` instead of `DEACTIVATION_TIMEOUT_MS`